### ISSUE TITLE
[12.0] Sale order total not update on UI when change Taxes in Order Lines

### ISSUE THREAD

**Issue by langbamit** [odoo/odoo#51935]
Impacted versions: 12.0-20200417
Steps to reproduce:
I'm install Odoo from [Docker](https://hub.docker.com/_/odoo) with commands:
```
docker run -d -e POSTGRES_USER=odoo -e POSTGRES_PASSWORD=odoo -e POSTGRES_DB=postgres --name db postgres:10


docker run -p 8069:8069 --name odoo --link db:db -t odoo:12.0
```
After that, I setup data with demo and install sale_management.
Add taxes in order lines of record SO006
Current behavior:
Total value of order not update

Expected behavior:
Total value of order should update

Note: I can't re-introduce in runbot, runbot return expected behavior.

**Comment by Yenthe666** [comment]
Hi @langbamit,

Thanks for your report! We do need reproducible steps on a runbot instance though.
Have you checked that there is no custom development on your server that impacts the computations? Have you tried updating your source code to the latest version?

Regards,
Yenthe

**Comment by langbamit** [comment]
1. Have you checked that there is no custom development on your server that impacts the computations? Yes, I run docker offficial images without modified anything (code, config,..).
2. Have you tried updating your source code to the latest version? Yes, My steps base on latest update Odoo 12.0 from [Docker Official Images](https://hub.docker.com/_/odoo). And I have a another setup with 12.0 branch from [Odoo github](https://github.com/odoo/odoo/tree/12.0), the result as the same above. What is the different things between my setup and runbot?

**Comment by Yenthe666** [comment]
Not sure and that's the real question. I haven't seen an issue like this anywhere honestly.
Without reproducable steps there is not much we can do though. You could try checking configurations or see if there has been no similar fix since the last official image.

**Comment by langbamit** [comment]
> Without reproducable steps there is not much we can do though.

Like I said, I tested in Docker official image with only 2 bash command like original post (Run in docker on Ubuntu 18.04 & Manjaro 20.0.1 with the same unexpected result).

Btw, change `python` to v3.7.7 fixed my issue, tested in github 12.0 branch and latest official image. Just install `python`  v3.7.7 and `pip install -r requirements.txt`, no extra setup.

**Comment by Yenthe666** [comment]
@d-fence this looks like one for you to check then I think?
